#!/usr/bin/env python
PACKAGE = "modelos_cinematicos"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Position_X",    double_t,    0, "Valor de X en el plano", 0.2,  -0.36, 0.36)
gen.add("Position_Z",    double_t,    0, "Valor de Z en el plano", 0.25,  -0.1,0.36)

gen.add("LOWER_elbow", bool_t , 0, "Orientaciòn deseada de codo",True)

gen.add("mov_inicial", bool_t , 0, "Marcar para enviar a la posición inicial",False)
gen.add("Ready", bool_t , 0, "Marcar para habilitar movimiento en simulaciòn",False)
gen.add("Dynamixel", bool_t , 0, "Marcar para habilitar movimiento robot real",False)

gen.add("Grafica_Python", bool_t , 0, "Dibuja la gràfica de Espacio de trabajo en Python",False)
gen.add("Workspace", bool_t , 0, "Dibujar espacio de trabajo",False)
gen.add("Borrar_workspace", bool_t , 0, "Borrar espacio de trabajo",False)
gen.add("Demo", bool_t , 0, "Hacer recorrido de limites con OPenbotV1",False)

exit(gen.generate(PACKAGE, "workspace_2gdl", "workspace_2g"))
